lib/sqlalchemy_ingres/_version.py

```diff
@@ -1,2 +1,2 @@
-version_tuple = __version_info__ = (0, 0, 5, 'dev1')
+version_tuple = __version_info__ = (0, 0, 6)
```

Comment:
unless you are doing a release I would leave this alone.

Reply:
Version reverted back to `0.0.5.dev1`